### PR TITLE
#34 Support regex validation of property values

### PR DESCRIPTION
**Pull request by vincent-zurczak**
Extend the syntax to verify property values against a regular expression.
BTW, I may ask one of my colleagues to perform reviews on this project, if no one else feels comfortable on the library and its usage. In which case we should discuss how this person could join the group.

### CODE REVIEW

File: lib/detik.bash [hunk 1]
```diff
@@ -38,18 +38,30 @@ try() {
 	property=""
 	expected_value=""
 	expected_count=""
+	verify_equality="true"
```
Comment (pascallibenzi) [inline lib/detik.bash:41]: Should we rename this verify_equality var to verify_strict_equality

File: README.md [hunk 2]
```diff
@@ -334,6 +337,65 @@ about the property names.
...
+# Assuming the status of the POD is "Running"...
+# ... then the following assertion will fail.
+verify "'status' matches 'running' for pods named 'nginx'"
+
+# Same for...
+verify "'status' matches '^running$' for pods named 'nginx'"
+
+# This is because the value returned by the client starts with an upper-case letter.
+# For case-insensivity operations with a regular expression, just use...
+DETIK_CASE_INSENSITIVE_PROPERTIES="true"
```
Comment (pascallibenzi) [inline README.md:387]: Should be renamed to DETIK_REGEX_CASE_INSENSITIVE_PROPERTIES

Comment (vincent-zurczak) [reply README.md]: Done. Thanks!

**Review by martin-schulze-vireso — APPROVED** [review]

**Comment by martin-schulze-vireso** [comment]
Hm, it seems I can only force thenmerge but the bats-detik team must approve. I am currently on my phone so I cannot dig deeper. I will investigate this later.

**Comment by vincent-zurczak** [comment]
In fact, my last commit was not signed.
I am trying to fix it.

**Comment by vincent-zurczak** [comment]
Ready to be merged. @martin-schulze-vireso 
Thanks!

**Pull request merged**